src/verify/VerificationStartScreen.tsx

```diff
@@ -265,28 +256,13 @@ function VerificationStartScreen({
         </TextButton>
       </View>
       <KeyboardSpacer />
-      <Dialog
-        title={t('phoneVerificationScreen.skip.title')}
-        isVisible={showSkipDialog}
-        actionText={t('phoneVerificationScreen.skip.confirm')}
-        actionPress={onPressSkipConfirm}
-        secondaryActionPress={onPressSkipCancel}
-        secondaryActionText={t('phoneVerificationScreen.skip.cancel')}
-        testID="PhoneVerificationSkipDialog"
-      >
-        {t('phoneVerificationScreen.skip.body')}
-      </Dialog>
```

Comment:
Just double checking, design / product were ok with dropping this confirm dialog?
Also, should we remove corresponding translations?

Reply:
Yes I double checked with them in Dm's. Its okay to drop